[Draft] organization doc restructure (#851)
RoboCupAtHome/RuleBook: CompetitionConcepts.tex

While they have to compete against each other during the competition, the members of the \AtHome{} league are expected to cooperate and exchange knowledge to advance technology together.
The \iterm{RoboCup@Home mailing list} as well as the \RR{} can be used to get in touch with other teams and to discuss league-specific issues such as rule changes, proposals for new tests, etc.
% Since 2007 there is also the \iterm{RoboCup@Home Wiki} (see~\refsec{sec:at_home_wiki}) which serves as a central place to collect information relevant for the @Home league.

> In addition, every team is expected to share relevant technical, scientific (and team-related) information in an scientific paper and on the team's website.

Reviewer: best to specify that we mean TDP here rather than any scientific paper. As TDP is not a scientific paper in most cases.
```suggestion
In addition, every team is expected to share relevant technical, scientific (and team-related) information in their TDP and on the team's website.
```
